chemicaltoolbox/chemfp/mol2fps.xml

```diff
@@ -3,8 +3,6 @@
     <!--parallelism method="multi" split_inputs="infile" split_mode="to_size" split_size="10000" shared_inputs="" merge_outputs="outfile"></parallelism-->
     <requirements>
         <requirement type="package" version="1.1p1">chemfp</requirement>
-        <requirement type="package" version="2016.03.3">rdkit</requirement>
-        <requirement type="package" version="2.3.2">openbabel</requirement>
```

Comment:
If you look at the chemfp package, openbabel and rdkit are already a dependency on chemfp, so you don't need to define them here. Even worse in this case chemfp was depending on a newer version of openbabel than we did specify here, ending up with two different openbabel versions in the same conda environment, which in the end confuses chemfp.